yarn-project/noir-protocol-circuits/src/crates/types/src/traits.nr

```diff
@@ -0,0 +1,7 @@
+trait Empty {
```

Comment:
nit: a short comment to say why this trait is needed would be good for future readers

Reply:
Done!